pkg/signup/service/signup_service_test.go

```diff
@@ -1235,13 +1239,25 @@ func (s *TestSignupServiceSuite) TestGetSignupByUsernameOK() {
 
 	us := s.newUserSignupComplete()
 	us.Name = service.EncodeUserIdentifier(us.Spec.IdentityClaims.PreferredUsername)
+	// Set the scheduled deactivation timestamp 1 day ago
+	us.Status.ScheduledDeactivationTimestamp = util.Ptr(v1.NewTime(time.Now().Add(-time.Hour * 24)))
```

Comment:
could you please explain why you set time 1 day ago? it's more logical to set the deactivation time in the future instead, right?

Reply:
It made more sense previously before I removed the DaysRemaining field from the response, now since we no longer need to test this functionality I can set the time in the future.  Updated.